docs/architecture-v2/bridge.md

```diff
@@ -0,0 +1,92 @@
+# v2 Architecture Bridge
+
+In order to migrate an existing MCP landscape to the new v2 architecture step by step, additional logic was added to the MCP operator, which allows it to switch between the old and the new logic for each component. The 'new logic' is usually just some bridge logic that transforms the v1 api type into the v2 api type and transforms the v2 api type's status back into the v1 format.
+
+The bridge is currently implemented for the following components:
+- `APIServer`
+
+## Architecture Configuration
+
+To configure for which components the bridge is enabled, the MCP operator now takes a new optional argument `--arch-config` which is expected to point to a YAML file that looks like this:
+```yaml
+apiServer:
+  version: v1
+  allowOverride: false
+# more components are to follow
```

Comment:
Is there a reason that we are creating a new seperate configuration file instead of putting this into the general configuration? Or isn't there one.
If there isn't one, would it make sense to create one fur all future purposes?